packages/sdk/tests/e2e/sharing/removeProtectedDataFromRenting.test.ts

```diff
@@ -109,17 +82,16 @@ describe('dataProtector.removeProtectedDataFromRenting()', () => {
       async () => {
         const protectedDataAddressMock = Wallet.createRandom().address;
         //create collection
-        const { collectionTokenId } = await dataProtector.createCollection();
+        await dataProtector.createCollection();
         await sleep(2000);
         //to simulate the error we won't add the protected data to the collection
         await expect(() =>
           dataProtector.removeProtectedDataFromRenting({
-            collectionTokenId: collectionTokenId,
             protectedDataAddress: protectedDataAddressMock,
           })
         ).rejects.toThrow(
           new WorkflowError(
-            'Failed to Remove Protected Data From Renting: Protected Data is not in collection.'
+            'This protected data does not exist in the subgraph.'
```

Comment:
Update the test description accordingly.
'should fail with protected data is not in collection error' does not seem to fit, does it?

Reply:
5eaf64eb2839e72c90b5bdc54624e2b0be1760d8

Reply:
Something seems off:

Test: 'should fail if the protected data does not exist'
Expected error: 'This protected data does not exist in the subgraph.'

Test: 'should fail with protected data is not in collection error'
Expected error: 'This protected data does not exist in the subgraph.' <-- Same as above, but it should be something else, right?